schema/proto3/gateway.proto

```diff
@@ -19,15 +19,18 @@ syntax = "proto3";
 package alcor.schema;
 
 option java_package = "com.futurewei.alcor.schema";
-option java_outer_classname = "AuxiliaryGateway";
+option java_outer_classname = "Gateway";
 
-enum AuxGatewayType {
+enum GatewayType {
     NONE = 0;
     ZETA = 1;
+    TGW = 2;
+    IGW = 3;
+    NGW = 4;
 }
 
-message AuxGateway {
-    AuxGatewayType aux_gateway_type = 1;
+message GatewayConfiguration {
+    GatewayType gateway_type = 1;
 
     // if aux_gateway_type == NONE, then id = empty string 
```

Comment:
comment update: aux_gateway_type to gateway_type

Reply:
fixed